src/OpenTelemetry/Metrics/MetricAggregators/IGaugeMetric.cs

```diff
@@ -14,23 +14,16 @@
 // limitations under the License.
 // </copyright>
 
-using System;
 using System.Collections.Generic;
-using System.Diagnostics.Metrics;
 
 namespace OpenTelemetry.Metrics
 {
-    // TODO: Need to determine what a Metric actually contains
-
-    internal struct Metric
+    internal interface IGaugeMetric : IMetric
     {
-        internal readonly string Name;
-        internal IDataPoint Point;
+        bool IsDeltaTemporality { get; }
+
+        IEnumerable<IDataPoint> Exemplars { get; }
```

Comment:
Interesting... I suppose exemplars do nearly match the definition of a DataPoint but with additional trace/span id. Is this just a prototype thing or are you planning on using `IDataPoint` to model exemplars long term?

Reply:
Good catch. I missed the trace/spanid in Exemplars!